bin/ebuild-helpers/prepall

```diff
@@ -19,7 +19,7 @@ fi
 prepallman
```

Comment:
On a related note, you probably want to ban all of `prep*` in EAPI 7.

Reply:
Sure, we can do that in a separate patch. I this patch, I'm only concerned with *strip commands due to overlap.